Client/Frontend/Fakespot/FakespotViewController.swift

```diff
@@ -181,7 +185,7 @@ class FakespotViewController: UIViewController, Themeable, UIAdaptivePresentatio
         _ = titleCenterYConstraint.priority(.defaultLow)
     }
 
-    private func recordTelemetry() {
+    private func recordDismissVCTelemetry() {
```

Comment:
Can we rename this to `recordDismissTelemetry`? We shouldn't use abbreviations in method.